app/assets/javascripts/components/students/containers/Overview.jsx

```diff
@@ -46,6 +47,7 @@ export class Overview extends React.Component {
         }
 
         <RandomPeerAssignButton {...this.props} />
+        { current_user.isAdvancedRole ? (<AddToWatchlistButton slug={course.slug} prefix={prefix} />) : null }
```

Comment:
It would be better to have this aligned next to, rather than beneath, the other button.

Reply:
@ragesoss for this should I just apply some CSS to make them aligned next to each other?